src/services/users.js

```diff
@@ -6,7 +7,17 @@ import {
   Role,
   sequelize,
   UserValidationStatus,
+  ActivityReport as ActivityReportModel,
+  ActivityReportCollaborator,
+  ActivityReportApprover,
+  ActivityRecipient,
+  Grant,
+  Goal,
```

Comment:
you can remove the Goal & Objective models from this import, it doesn't look like they end up being used

Reply:
Will do.